### ISSUE TITLE
Keyboard access

### ISSUE THREAD

**Issue by VirginiaBalseiro** [dokieli/dokieli#462]
Critical: toolbar and slash menu are not accessible via keyboard (not tabbable) and should be focused when it appears.
Improvement: tab into menu button first (currently need to tab sequentially through the entire page before getting to the menu, it should be easier to access)
Improvement: focus on editor area when entering on Edit button.
Improvement: Do "skip to x", Esc, ctrl-something-something to Edit mode. Add this to documentation as well
